spec/dummy/app/themes/default/assets/stylesheets/layout/_topic-navigation.scss

```diff
@@ -24,7 +24,9 @@
 
 }
 
+#thredded_new_topic .topic-navigation--public,
```

Comment:
Avoid using id selectors<br>Selector `thredded_new_topic` should be written in lowercase with hyphens<br>Selector `thredded_topics_index` should be written in lowercase with hyphens<br>Selector `thredded_new_private_topic` should be written in lowercase with hyphens<br>Selector `thredded_private_topics_index` should be written in lowercase with hyphens